angular-workspace/components/list-button/list-button.stories.ts

```diff
@@ -12,7 +12,7 @@ const meta: Meta<ListButtonArgs> = {
   title: "List Button",
   decorators: [
     moduleMetadata({
-      declarations: [TrustHtmlPipe],
+      imports: [TrustHtmlPipe],
```

Comment:
In angular 19 zijn directives, components en pipes default standalone.

Reply:
Interessant.

Heeft dit nog gevolgen voor de afnemers?

Reply:
Ik probeer 'iets' te bedenken.  Maar er komt niets. 🫣

Reply:
Op de [website](https://www.dso-toolkit.nl/67.1.0/voor-developers/web-components/angular-bindings/) lees ik:
```
Als er gebruik wordt gemaakt van Angular heeft deze methode de voorkeur. De Angular bindings bestaat uit een gegenereerde NgModule met Angular Componenten.
```